gtest/src/main.rs

```diff
@@ -38,15 +38,20 @@ struct Opts {
     pub skip_memory: bool,
     /// JSON sample file(s) or dir
     pub input: Vec<std::path::PathBuf>,
+    /// A level of verbosity
```

Comment:
Do I understand coorectyl that this is not a level actually but just a flag if the output should be verbose?

Reply:
yep, maybe will add levels in the future